src/pages/benchmarks/index.tsx

```diff
@@ -12,21 +12,50 @@ import $button from "../../components/design_system/button/Button.module.css";
 import noBenchmarks from "../../public/img/no-benchmarks.png";
 import SearchIcon from "@mui/icons-material/Search";
 
+type SortProperty = "first_name";
+type SortDirection = "asc" | "desc";
+
 function Benchmarks() {
   const [isPara, setIsPara] = useState(false);
-  const { data: tasks, isLoading } = trpc.para.getMyTasks.useQuery();
+
+  const [sortProperty, setSortProperty] = useState<SortProperty>("first_name");
+  const [sortDirection, setSortDirection] = useState<SortDirection>("asc");
+
+  const { data: tasksData, isLoading, error } = trpc.para.getMyTasks.useQuery();
```

Comment:
Is "error" within data going to be accessed, or is this for troubleshooting of some sort?

Reply:
We can remove it for now

Reply:
sounds good